easybuild/easyconfigs/c/cURL/cURL-7.54.1-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.54.1'
+
+homepage = 'http://curl.haxx.se'
+
+description = """
+ libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
+ LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+ libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://curl.haxx.se/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+osdependencies = [
+    ('libssh2-devel'),
```

Comment:
Hmm, this part is new...
I'm not a big fan of requiring more OS dependencies than strictly needed.

What's the big win of building `cURL` with `--with-libssh2` @JackPerdue?